sdks/go/pkg/beam/util/shimx/generate_test.go

```diff
@@ -76,7 +76,7 @@ func TestTop_ProcessImports(t *testing.T) {
 		{name: "shim", got: &Top{Shims: []Func{{Name: "emit"}}}, want: []string{ReflectxImport, "context", "keepit", "fmt", "io", "unrelated"}},
 		{name: "iter&emit", got: &Top{Emitters: []Emitter{{Name: "emit"}}, Inputs: []Input{{Name: "iter"}}}, want: []string{ExecImport, TypexImport, "keepit", "unrelated"}},
 		{name: "functions", got: &Top{Functions: []string{"func1"}}, want: []string{RuntimeImport, "context", "keepit", "fmt", "io", "unrelated"}},
-		{name: "types", got: &Top{Types: []string{"func1"}}, want: []string{RuntimeImport, "context", "keepit", "fmt", "io", "unrelated"}},
+		{name: "types", got: &Top{Types: []string{"func1"}}, want: []string{SchemaImport, RuntimeImport, "context", "keepit", "fmt", "io", "unrelated"}},
```

Comment:
A test with a type prefixed with `"beam."` could be good to test that behavior works. Same as this test, but it could say `beam.func1` and would expect `SchemaImport` to not be present.

Reply:
Ack.